src/components/About.vue

```diff
@@ -146,6 +146,16 @@ $coolBlue: rgb(66, 145, 235);
 
   }
 }
+.usa-nav__primary button[aria-expanded=false] span::after {  //removes those little squares that were appearing by Recent, Visualizations, About
+    background: url([object Object]) no-repeat center/contain;
+    display: inline-block;
+    height: 0px;
+    width: 0px;
+    content: "";
```

Comment:
Curious - what does this `content: ""` do?

Reply:
Good catch! Looks like nothing. Removed.